rules/falco_rules.yaml

```diff
@@ -227,7 +227,7 @@
 - list: deb_binaries
   items: [dpkg, dpkg-preconfigu, dpkg-reconfigur, dpkg-divert, apt, apt-get, aptitude,
     frontend, preinst, add-apt-reposit, apt-auto-remova, apt-key,
-    apt-listchanges, unattended-upgr, apt-add-reposit, apt-config, apt-cache
+    apt-listchanges, unattended-upgr, apt-add-reposit, apt-config, apt-cache, apt.systemd.dai
```

Comment:
Is this supposed to be `apt.systemd.daily`?

Reply:
Yup, but all the items' lengths are capped to 15 characters...